data/json/items/melee/unarmed_weapons.json

```diff
@@ -84,10 +84,10 @@
   },
   {
     "type": "GENERIC",
-    "id": "knuckle_steel",
+    "id": "knuckle_scrap",
```

Comment:
Please also add a migration entry to `data/json/items/migration.json` to convert `knuckle_steel` items in old saves to `knuckle_scrap`.

Reply:
There's also `"overlay_wielded_knuckle_steel"` in `data/mods/Chibi_Ultica/mod_tileset.json`.

Reply:
Oh, gotcha, thanks for notifying me of that. I’ll get right to it.

Reply:
Why not leave the item ids the same and just change the displayed name? Nothing breaks that way.

Reply:
Seconded, don't change the ID, it's not displayed to the player anyway, so having it not match the name is not a big deal.